napari/_qt/widgets/qt_viewer_status_bar.py

```diff
@@ -15,7 +22,49 @@ class ViewerStatusBar(QStatusBar):
     def __init__(self, parent: '_QtMainWindow') -> None:
         super().__init__(parent=parent)
 
-        self.showMessage(trans._('Ready'))
+        main_widget = QWidget()
+
+        layout = QHBoxLayout()
+
+        self._status = QLabel('Ready')
+        self._status.setMinimumSize(0, 16)
+        self._status.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Maximum)
```

Comment:
To help limit the spacing as much as possible, I would set content margins to 0 on all of these inner widgets **and** on the layout itself e.g.

```suggestion
        self._status.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Maximum)
        self._status.setContentsMargins(0, 0, 0, 0)
```